convert_hf_to_gguf.py

```diff
@@ -1161,7 +1161,7 @@ def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None) -> Iter
         # So we rearrange them here,, so that we have n_head query weights
         # followed by n_head_kv key weights followed by n_head_kv value weights,
         # in contiguous fashion.
-        # ref: https://github.com/jploski/ggml/blob/falcon40b/examples/falcon/convert-hf-to-ggml.py
+        # ref: https://github.com/jploski/ggml/blob/falcon40b/examples/falcon/convert_hf_to_gguf.py
```

Comment:
This one is part of a link, and so I think it should be left intact.

Reply:
Note that the original ends with `-ggml.py`, not `-gguf.py`, which did not exist at the time.